cas-server-webapp/src/main/webapp/WEB-INF/view/jsp/protocol/3.0/casServiceValidationSuccess.jsp

```diff
@@ -49,9 +49,6 @@
                         <cas:${fn:escapeXml(attr.key)}>${fn:escapeXml(attrval)}</cas:${fn:escapeXml(attr.key)}>
                     </c:forEach>
                 </c:forEach>
-                <cas:isFromNewLogin>${fn:escapeXml(isFromNewLogin)}</cas:isFromNewLogin>
```

Comment:
Agreed. They are different, and as you say, clients will need to look up the attribute anyway. I just don't want to impose major protocol changes. Even the bits that are introduced in this pull, I feel like we can simply dub as extension and optional attributes [as the protocol describes] and as I mentioned, I dont foresee any practical benefits. This is just a logical separation with minimum gain, the client itself can make that separation via parsers, etc.